contrib/fwupd.spec.in

```diff
@@ -181,6 +183,30 @@ This provides the optional package which is only required on hardware that
 can be flashed using flashrom. It is probably not required on servers.
 %endif
 
+%package qubes-dom0
```

Comment:
Similarly to the Debian case I think we want to special case the generation of the `qubes` binary package.  This won't go into real Fedora archives.

Reply:
Completely agree.

Reply:
There is a possibility to generate separate `fedora-qubes` and `debian-x86_64-qubes` containers that build qubes packages. Standard containers don't provide qubes DEBs and RPMs.